### ISSUE TITLE
Support for load balancer?

### ISSUE THREAD

**Issue by asthomasdk** [terraform-providers/terraform-provider-scaleway#294]
HI guys, 

I am wondering if support for your new load balancer will be included in the terraform provider. I don't see it listed anywhere in the roadmap for version 2. Or I may be overlooking it ;-) 

Thanks!

/Thomas

**Comment by kindermoumoute** [comment]
Yes! This is in the pipeline, see https://github.com/terraform-providers/terraform-provider-scaleway/pull/284

**Comment by asthomasdk** [comment]
Excellent. 

Would be great to get this soon - as I am trying to convert a set of Rancher 2 AWS terrform scripts to work with Scaleway ;-)
